<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN">
<html>
	<head>
		<script type="text/javascript" src="https://code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=" crossorigin="anonymous"></script>
		<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
		<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
		<script async defer src="https://buttons.github.io/buttons.js"></script>
		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" type="text/css">
		<link href="https://fonts.googleapis.com/css?family=Roboto:100,300" rel="stylesheet">
		<link rel="stylesheet" src="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css">
		<link rel="stylesheet" href="../css/style.css" type="text/css">
		<title>
			Michael Shaffer's Projects
		</title>
	</head>
	<body>
		<div class="container">
			<div class="row">
				<div class="col-md-2">
					<img class="img-responsive center-block" src="../images/headshot_2.png">
				</div>
				<div class="col-md-8 text-left text-uppercase">
					<h1>
						Michael Shaffer
					</h1>
					<h4>
						Microbiome bioinformatics scientist
					</h4>
				</div>
				<div class="col-md-2 buttons">
					<div class="text-right">
						<a class="github-button" href="https://github.com/shafferm" data-style="mega" aria-label="Follow @shafferm on GitHub">Follow @shafferm</a>
					</div>
					<div class="text-right">
						<a href="https://twitter.com/mikerobiome" class="twitter-follow-button" data-size="large" data-show-count="false">Follow @mikerobiome</a>
					</div>
				</div>
			</div>
			<hr>
			<div class="row">
				<div class="col-md-12">
					<h2>
						My Projects
					</h2>
				</div>
				<div class="col-md-12">
					<p>
						I have been involved in research since 2011 and have had the oppurtunity to work on a variety of projects.
						<br><br>
						As an undergraduate I did research in the lab of Catherine Putonti at Loyola University Chicago. Here I primarily studied phage-host interactions via the CRISPR system. I developed a tool for use in the lab to match CRISPR spacer seqeunces from microbial genomes to phage genomes and generate a interaction network.
						<br><br>
						As a graudate student I studied in Catherine Lozupone's lab at University of Colorado Anschutz Medical Campus where I worked on developing methods 1) for correlation network analysis of compositional data such as 16S rRNA amplicon sequencing and untarged LC/MS metabolomics and 2) connecting the genomic content of microbes to compounds they could potential produce/consume. In addition to these bioinformatics software development projects I also applied these methods, and others, to researching the effect of HIV on the human gut microbiome.
						<br><br>
						I am currently a postdoc in Kelly Wrighton's lab at Colorado State University. I've had the oppurtunity to work on a wide variety of projects in the Wrighton lab. DRAM is my primary software development project. It is a tool that will annotate bacterial genomes/metagenomes and tell you what functions they are potentially capable of. I built all of the software while the metabolism and other genome function was curated by a team of experts led by my co-first author Mikayla Borton. My primary biologly focused work is in studying the effects Salmonella infection on the gut microbial community in mice. This work invovles collaboration with the Ahmer and Wysocki labs at Ohio State University and I analyze the sequencing based 'omics and lead the integration between mass spec based 'omics and the sequencing based 'omics. Additionally I've gotten to help out with other projects doing statistical analysis as well as bioinformatics pipeline development in environments including the human gut, global river systems, wetland sediments and permafrost soils.
						<br><br>
						Here is some more detail on some of the projects I'm most proud of.
					</p>
				</div>
				<div class="col-md-12">
					<h3>
						DRAM: Annotating and summarizing functional content of microbial genomes/metagenomes
					</h3>
					<div class="col-md-3">
						<a href="http://github.com/shafferm/DRAM">
							<img class="img-responsive white-background center-block" src="../images/DRAM_logo_large.png">
						</a>
					</div>
					<div class="col-md-9">
						<p>
							DRAM more than a genome annotation tool. While it performs this function, and does very well at it in comparison to similar software, what really sets DRAM apart is the ability to summarize the functions that are present in a genome/metagenome in an easy to understand way. It distills this information to generate three levels for users to examine. The raw contains all annotations for all genes from all databases. The distillate gives an overview of all the functional genes which are present across all genomes. The product gives an interative heatmap allowing the user to explore a high level overview of the functions a acollection of genomes are capable of. <a href="../examples/dram/index.html">You can check out what the product looks like by clicking here.</a>
							<br><br>
							In order to make DRAM available to a wider audience <a href="https://narrative.kbase.us/#catalog/modules/kb_DRAM">we have also developed a series of KBase apps</a>. KBase is a cyberinfrastructure developed by the Department of Energy with the goal of making 'omics analysis available to everyone. You can read more about the project at <a href="www.kbase.us">www.kbase.us</a>. The DRAM KBase apps allow users to access DRAM through a web interface and without needing access to powerful servers. The DRAM KBase apps also allow for integration into the larger KBase system of tools including using DRAM annotations in flux balance analysis simulations.
							<br><br>
							DRAM recieved over 20 citations in the first year it was published. <a href="https://doi.org/10.1093/nar/gkaa621">You can check it out in Nucleic Acids Research</a>. We also had the opportunity to present DRAM as part of the Ohio State University Center of Microbiome Science webinar series <a href="https://u.osu.edu/coms/webinar-resources/">which you can watch online here.</a> DRAM development continues <a href="https://github.com/shafferm/DRAM">on it's github page</a> where there is also <a href="https://github.com/shafferm/DRAM/wiki">a wiki full of helpful documentation</a>.
						</p>
					</div>
				</div>
				<div class="col-md-12">
					<h3>
						2020 Multiscale Microbial Dynamics Modeling Summer School
					</h3>
					<div class="col-md-3">
						<a href="https://pnnl.cvent.com/events/multiscale-microbial-dynamics-modeling-summer-school/agenda-7706656ec1004be0ba368947c10c375b.aspx">
							<img class="img-responsive white-background center-block" src="https://www.pnnl.gov/sites/default/files/styles/hero_1600x1200/public/media/image/Banners_Event_HP_1600x1200.png">
						</a>
					</div>
					<div class="col-md-9">
						<p>
							In 2020 I had the opportunity to act as an instructor for the 2020 DOE Environmental Molecular Sciences Laboratory summer school. This is a program put on each summer by EMSL with a different theme every year. The 2020 edition focused on microbial dynamics modeling. Ordinarily this program is run in person but this was not possible due to the COVID-19 pandemic. As a result it was taught online and the general lectures were opened up to anyone who wanted to attend while the small group projects were run online. I taught a module about genome annotation and how DRAM can be used to annotate gene function. <a href="https://www.youtube.com/watch?v=zjto9kbBPzQ&list=PL6ecvCJPoOtmshHIV0Gpzy8ibxiJBgM94">The talk I gave is available on youtube</a>. Additionally I build the KBase narratives that were used by the students and led a student group during the event. As a cap to this work I presented a talk at the 2020 AGU fall meeting about the discoveries we were able to make in collaborations with the students who participated.
						</p>
					</div>
				</div>
				<div class="col-md-12">
					<h3>
						SCNIC: Building and analyzing correlation networks made from compositonal data
					</h3>
					<div class="col-md-4">
						<a href="http://github.com/shafferm/SCNIC">
							<img class="img-responsive white-background center-block" src="../images/SCNIC_sematic_horizontal_small.png">
						</a>
					</div>
					<div class="col-md-8">
						<p>
							Some words about SCNIC. Mention preprint. Mention q2-SCNIC.
						</p>
					</div>

				</div>
				<div class="col-md-12">
					<h3>
						AMON: Determining potential metabolite origin from microbial genomes
					</h3>
					<p>
						Some words about AMON. Link to paper. Link to github.
					</p>
				</div>
			</div>
		</div>
	</body>
</html>
